cfme/infrastructure/provider/__init__.py

```diff
@@ -167,7 +167,7 @@ def vm_name(self):
         return version.pick({
             version.LOWEST: "VMs",
             '5.5': "VMs and Instances",
-            '5.7': "Virtual Machines"})
+            '5.7.1': "Virtual Machines"})
```

Comment:
What am I missing here, these strings look the exact same?

I think you can go further here though, removing the `version.LOWEST` and `5.5` keys from the dictionary, just leaving 5.7 (which should be selected for 5.7.1 as well)